public_html/layouts/basic/modules/Vtiger/resources/dashboards/Widget.js

```diff
@@ -1386,24 +1386,27 @@ jQuery.Class('Vtiger_Widget_Js', {
 		image.src = base64Image;
 		return image;
 	},
+	printImage(imgEl, title, width, height) {
+		const print = window.open('', 'PRINT', 'height=' + height + ',width=' + width);
+		print.document.write('<html><head><title>' + title + '</title>');
+		print.document.write('</head><body >');
+		print.document.write($('<div>').append(imgEl).html());
+		print.document.write('</body></html>');
+		print.document.close(); // necessary for IE >= 10
+		print.focus(); // necessary for IE >= 10
+		setTimeout(function () {
+			print.print();
+			print.close();
+		}, 1000);
+	},
 	registerHeaderButtons: function registerHeaderButtons() {
 		const container = this.getContainer();
 		const header = container.find('.dashboardWidgetHeader');
 		const downloadWidget = header.find('.downloadWidget');
 		const printWidget = header.find('.printWidget');
 		printWidget.on('click', (e) => {
 			const imgEl = this.getChartImage();
-			const print = window.open('', 'PRINT', 'height=400,width=600');
-			print.document.write('<html><head><title>' + header.find('.dashboardTitle').text() + '</title>');
-			print.document.write('</head><body >');
-			print.document.write($('<div>').append(imgEl).html());
-			print.document.write('</body></html>');
-			print.document.close(); // necessary for IE >= 10
-			print.focus(); // necessary for IE >= 10
-			setTimeout(function () {
-				print.print();
-				print.close();
-			}, 1000);
+			this.printImage(imgEl, header.find('.dashboardTitle').text(), 600, 400);
```

Comment:
600 >100%?

Reply:
get screen resolution and open window in full screen?